pages/home.js

```diff
@@ -1,19 +1,78 @@
 import PropTypes from 'prop-types'
 import en from '../locales/en'
 import fr from '../locales/fr'
+import HomeCardContainer from '../components/HomeCardContainer'
+import TextInput from '../components/TextInput'
 
 import { fetchContent } from '../lib/cms'
 
 export default function Home(props) {
   /* istanbul ignore next */
   const t = props.locale === 'en' ? en : fr
+
+  const handleCreateSubmit = (e) => {
+    e.preventDefault()
+    console.log(e.target.owner.value)
+
+    //TODO: Make call to back end to get random room id.
+    //TODO: Redirect user to that room id.
+    //  router.push({
+    //    pathname: "/room/1",
+    //    query: {q: "test"},
+    //  })
+  }
+
+  const handleJoinSubmit = (e) => {
+    e.preventDefault()
+  }
+
   return (
     <div
+      data-testid="homeContent"
       id="homeContent"
-      className="container mx-auto px-6 mt-5 bg-slate-300 p-8"
+      className="container grid grid-cols-1 gap-y-5 mx-auto sm:flex sm:justify-around"
     >
-      <h1>{props.content.header}</h1>
-      <p>{props.content.paragraph}</p>
+      <HomeCardContainer title={t.createRoomTitle} desc={t.createRoomDesc}>
+        <form
+          onSubmit={handleCreateSubmit}
+          className="flex flex-col justify-between h-full"
+        >
+          <TextInput
+            id="owner"
+            label={t.createRoomLabel}
+            placeholder={t.createRoomPlaceholder}
+          />
+          <button
+            type="submit"
+            className="bg-green-500 hover:bg-green-600 font-bold py-2 px-4 rounded max-h-10 mt-12 focus:drop-shadow focus:ring-2 focus:ring-gray-600"
+          >
+            {t.createRoomButton}
+          </button>
```

Comment:
```suggestion
          <div>
            <button
              type="submit"
              className="w-max font-display text-white bg-[#318000] hover:bg-[#1D4D00] active:bg-[#102900] py-3 px-5 rounded mt-12 focus:drop-shadow focus:ring-2 focus:ring-gray-600 border border-[#458259] text-[22px] leading-8 [text-shadow:1px_2px_0px_#333]"
            >
              {t.createRoomButton}
            </button>
          </div>
```

The above suggestion would bring the button more in line with the current figma designs. Since I don't know what our timeline is for using the design system, I used arbitrary values in the above styling rather than suggest you add custom colors (or the custom text-shadow functionality, why doesn't Tailwind have this by default? 👎 ), but those can always be added later.

This uses the colors straight from the GC Design Library.